Some moving around of files + Dockerfile

Bartman0 opened the pull request:
I moved around some files to make it more package like.
And I added a Dockerfile to package it all and make it runnable from Docker.

tdorssers commented:
thanks for this update, i made some changes:
- reverted cache.json relative parent path lookup, this didn't work on windows
- replaced pathlib by pkgutil to maintain py2/3 and windows compatibility
- entrypoint.sh was missing run_menu() and run_gui()
- added importing requests.exceptions to __init__.py, to fix name errors in gui.py
- reverted changes to gui.py to fix name errors: eg. teslapy.VehicleError or teslapy.RequestException

Bartman0 commented:
Cool, I’ll pull the latest version.
Nice work on the package.

From: Tim Dorssers <notifications@github.com>
Reply to: tdorssers/TeslaPy <reply@reply.github.com>
Date: Tuesday, 24 December 2019 at 13:54
To: tdorssers/TeslaPy <TeslaPy@noreply.github.com>
Cc: Richard Kooijman <Richard.Kooijman@inergy.nl>, Author <author@noreply.github.com>
Subject: Re: [tdorssers/TeslaPy] Some moving around of files + Dockerfile (#3)


thanks for this update, i made some changes:

  *   reverted cache.json relative parent path lookup, this didn't work on windows
  *   replaced pathlib by pkgutil to maintain py2/3 and windows compatibility
  *   entrypoint.sh was missing run_menu() and run_gui()
  *   added importing requests.exceptions to init.py, to fix name errors in gui.py
  *   reverted changes to gui.py to fix name errors: eg. teslapy.VehicleError or teslapy.RequestException

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/tdorssers/TeslaPy/pull/3?email_source=notifications&email_token=ABRZ7YSB7LVCFZYN5U45PU3Q2IBBZA5CNFSM4J6JOV72YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEHTFJRA#issuecomment-568743108>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABRZ7YUTU43EDXZAWCCZHVTQ2IBBZANCNFSM4J6JOV7Q>.

Richard Kooijman
Advanced Analytics Architect
Inergy
Pelmolenlaan 2 • 3447 GW Woerden • T: +31 348 457 666 • M: +31 631 673 387 • www.inergy.nl
Magazine Inergy | Heb jij het al gelezen?
Kerstgeschenk | Fijne feestdagen
Blog | Hoe maak je een effectief dashboard?
